pkg/install/start_vms.go

```diff
@@ -5,70 +5,27 @@ package install
 
 import (
 	"context"
-	"strings"
 
-	mgmtcompute "github.com/Azure/azure-sdk-for-go/services/compute/mgmt/2019-03-01/compute"
 	"golang.org/x/sync/errgroup"
 
 	"github.com/Azure/ARO-RP/pkg/util/stringutils"
+	"github.com/Azure/ARO-RP/pkg/util/virtualmachines"
 )
 
 // startVMs checks cluster VMs power state and starts deallocated and stopped VMs, if any
 func (i *Installer) startVMs(ctx context.Context) error {
 	resourceGroupName := stringutils.LastTokenByte(i.doc.OpenShiftCluster.Properties.ClusterProfile.ResourceGroupID, '/')
-	vms, err := i.virtualmachines.List(ctx, resourceGroupName)
+	vmsToStart, err := virtualmachines.ListStopped(ctx, i.virtualmachines, resourceGroupName)
```

Comment:
I would pause with something like this.
This clashes with https://msazure.visualstudio.com/AzureRedHatOpenShift/_workitems/edit/7687268
From now customer should be able to power off machines. 
Currently, we do not support this in our API.
We might want to stop alerting if machines are powered off, as we should not have issues now.
In the future, this should come part of API schema and we allow the customer to do this directly. 

Originally this was an issue that cluster was not able to be power off due to certs. This is not an issue anymore.

Reply:
What do you think about:
- Not emitting a new metric
- Not notifying the user
- Not starting the VMs again
- Just returning early so we don't raise API Server Healthz alert (until powering off machines is fully supported)

The customer likely wouldn't raise an issue if they deallocated the VM themselves, but we could make a note in our SOP to check for that in case they did.

Reply:
> We might want to stop alerting if machines are powered off, as we should not have issues now.

Yes, that's the goal of this story. We need to be able to detect easily if machines have been powered off in a cluster, and how many of the total is powered off (all or none?)

I agree with Angus's suggestion, I think a good start is to emit a basic metric that has the number of powered off machines in a given cluster.

Reply:
Rather than emitting just a total, I have it emitting for each one, with the ID and the specific power state. That way we can get the total in a dashboard and also know which specific VMs are turned off.